magento/magento2
# [MFTF] Refactoring AdminMassOrdersHoldOnCompleteTest

### Description (*)
The test is refactored to improve execution time

### Manual testing scenarios (*)
1. Create an order
2. Invoice and ship the order
3. Go to the Orders Grid
5. Try to hold the order via mass update
6. Verify the result

### Related Pull Requests
https://github.com/magento/magento2/pull/31509

### Resolved issues:
1. [x] resolves magento/magento2#31515: [MFTF] Refactoring AdminMassOrdersHoldOnCompleteTest
